src/Components/IndexScene/VariableLayoutScene.tsx

```diff
@@ -99,6 +99,16 @@ export class VariableLayoutScene extends SceneObjectBase<VariableLayoutSceneStat
                     ) : null;
                   })}
                 </div>
+                {!currentFiltersMatchReference && (
```

Comment:
Just do you're aware, this can be potentially displayed if the reference filters are not defined.


```typescript
public currentFiltersMatchReference() {
    const referenceLabelsDefined = this.state.referenceLabels && this.state.referenceLabels.length > 0;
    return (
      !referenceLabelsDefined || ...
```

Reply:
Yeah, although currently if not embedded it will redirect to the service selection scene, but embedded without reference would display this button (which does nothing)